src/smart-components/group/group.scss

```diff
@@ -0,0 +1,3 @@
+.edit-group-button {
+  float: right;
```

Comment:
You shouldn't use floats, if you want to position the edit button you should use [level](https://patternfly-react.surge.sh/documentation/react/layouts/level) layout.

Reply:
Can you remove this file? Looks like it is no longer needed.